src/ert/simulator/simulation_context.py

```diff
@@ -68,12 +68,15 @@ def _run_forward_model(
     ):
         queue_evaluators = [
             partial(
-                job_queue.stop_long_running_jobs,
+                job_queue.stop_long_running_realizations,
                 ert.ert_config.analysis_config.minimum_required_realizations,
             )
         ]
 
-    asyncio.run(job_queue.execute(evaluators=queue_evaluators))
+    try:
+        asyncio.run(job_queue.execute(evaluators=queue_evaluators))  # type: ignore
+    except Exception:
+        job_queue.kill_all_jobs()
```

Comment:
This trickles down to an  asyncio.create_task() in realization_state.py. Will that work outside the async context?

Reply:
That's what I don't know :man_shrugging:

Reply:
It won't work. You'll get "RunTimeError: no running event loop"

Reply:
removed!